etcdctl/ctlv2/command/error.go

```diff
@@ -30,10 +32,15 @@ const (
 	ExitClusterNotHealthy
 )
 
-func handleError(code int, err error) {
-	fmt.Fprintln(os.Stderr, "Error: ", err)
-	if cerr, ok := err.(*client.ClusterError); ok {
-		fmt.Fprintln(os.Stderr, cerr.Detail())
+func handleError(code int, err error, isJSON bool) {
+	if isJSON {
+		b, _ := json.Marshal(etcderr.NewError(code, err.Error(), 0))
```

Comment:
do a type assertion against this https://github.com/coreos/etcd/blob/master/client/keys.go#L57 and simply marshal it to json and print it out